### PR TITLE
Adding xdebug PHP debugger and profiler

### PR DESCRIPTION
**Pull request by asafo**

### CODE REVIEW

File: srcpkgs/xdebug/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'xdebug'
+pkgname=xdebug
+version=2.3.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="php-devel autoconf"
```
Comment (xtraeme) [inline srcpkgs/xdebug/template]: Move `php-devel` to `makedepends` and then remove `php`.

File: srcpkgs/xdebug/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+pre_configure()
+{
+	phpize
+}
+
+do_install()
+{
+	cd ${wrksrc}
```
Comment (xtraeme) [inline srcpkgs/xdebug/template]: You can remove this line, this is done already.

File: srcpkgs/xdebug/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'xdebug'
+pkgname=xdebug
+version=2.3.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="php-devel autoconf"
+makedepends="php"
+short_desc="PHP Debugger"
+maintainer="Asaf Ohayon <asaf.tvn@gmail.com>"
+license="The Xdebug License, version 1.01"
```
Comment (xtraeme) [inline srcpkgs/xdebug/template]: Custom license but you are not installing it with `vlicense`, please fix.

File: srcpkgs/xdebug/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+homepage="http://xdebug.org"
+distfiles="http://xdebug.org/files/$pkgname-$version.tgz"
+checksum="b27bd09b23136d242dbc94f4503c98f012a521d5597002c9d463a63c6b0cdfe3"
+
+
+pre_configure()
```
Comment (xtraeme) [inline srcpkgs/xdebug/template]: Please change functions to use our common style:

```
func() {
   ...
}
```

File: srcpkgs/xdebug/template (hunk 3 above)
Comment (asafo) [inline srcpkgs/xdebug/template]: Thanks xtraeme, don't understand this comment, what is vlicense ?, how to fix

Comment (pullmoll) [inline srcpkgs/xdebug/template]: See https://github.com/voidlinux/void-packages/blob/master/Manual.md#global_funcs at the end. `vlicense <file>` is used to install something like `COPYING`, `LICENSE` or similar. You have to determine the actual `<file>` which contains the license, or even have to extract it from some source file.

**Pull request closed without merging**